schema.sql

```diff
@@ -68,4 +68,16 @@ CREATE TABLE visits (
 	vet_id INT REFERENCES vets(id),
 	animal_id INT REFERENCES animals(id),
 	date_of_visit DATE NOT NULL
-);
\ No newline at end of file
+);
+
```

Comment:
- [ ] [OPTIONAL] There is a typo in the 15 line that I missed previously 
<img width="220" alt="Screenshot 2023-05-09 at 18 55 48" src="https://github.com/blueberry1312/vet-clinic-database/assets/61790587/5460dd91-a0e2-4e54-be2b-a84e5bc08041">